godot_project/editor/input_handlers/molecular_structures/add_posing_atom_input_handler.gd

```diff
@@ -130,6 +130,7 @@ func _update_candidates_if_needed() -> void:
 	if not _candidates_dirty:
 		return
 	_candidates.clear()
+	_get_rendering().atom_autopose_preview_set_candidates(_candidates)
```

Comment:
wouldn't be better to move this line of code to line 140 right before the early return to avoid calling this method 2 times in the case a valid collection of candidates is set?

Reply:
Oh right, absolutely